hepcrawl/spiders/arxiv_spider.py

```diff
@@ -100,6 +100,10 @@ def parse_node(self, response, node):
             license_url=node.xpath('.//license//text()').extract_first()
         )
         record.add_value('license', license)
+        record.add_value(
+            'copyright_material',
+            'preprint',
+        )
```

Comment:
indeed, `copyright` should only be present if there is a copyright statement. This just says: the copyrighted material we are talking about is the preprint, without saying anything about the copyright.